examples/hawkbit-example-mgmt-simulator/src/main/java/org/eclipse/hawkbit/mgmt/client/scenarios/ConfigurableScenario.java

```diff
@@ -316,31 +316,38 @@ private void waitUntilRolloutIsReady(final Long id) {
 
     private void createDistributionSets(final Scenario scenario) {
         LOGGER.info("Creating {} distribution sets", scenario.getDistributionSets());
+        final int pages = (scenario.getDistributionSets() / PAGE_SIZE) >= 1
```

Comment:
The page calculation is broken when you slightly exceed the page size. For example: 506 or 501 distribution sets will result in a page size of 1 so in the end this code creates only 500 entities and not the expected amount.

Same happens to the other entities e.g. targets so I won't comment it there as well